### PR TITLE
DAH-1156 Prevent overflow of footer links

### PR DESCRIPTION
**Pull request by emilyjablonski**
[DAH-1156](https://sfgovdt.jira.com/jira/software/c/projects/DAH/boards/122?modal=detail&selectedIssue=DAH-1156)

Uptakes latest Bloom which includes a fix to allow for overflowing links in the `SiteFooter` component. Due to the breaking changes in the AdditionalFees component (which has a PR out waiting on a string question), I have just commented that out for now. Also adds the new stylesheets coming in from the Bloom theming work. To test, ensure the footer links do not overflow inappropriately on desktop and mobile screens.

### CODE REVIEW

File: app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsApplicationDate.test.tsx.snap [hunk 1]
```diff
@@ -35,7 +35,7 @@ exports[`ListingDetailsApplicationDate displays Applications Closed when listing
     data-test-id={null}
   >
     <svg
-      fill="#E41D3D"
+      fill="var(--bloom-color-red-700)"
```
Comment (emilyjablonski) [inline app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsApplicationDate.test.tsx.snap:38]: New theming starting to come through 😁

**Comment by christine-sfg** [comment]
I noticed the links are hard to read in Filipino, but actually are an improvement over what's currently in prod imo since at least each group of text stays together. I think we may still want to check with @yindipei though if they have any suggestions on wrapping or placement.
image

**Comment by yindipei** [comment]
I agree with Christine. Ideally, words that make up one link shouldn't wrap. In the case of Filipino, `Pagwawaksi ng Pananagutan (Disclaimer)` should all be on the same line. 

Additionally, for `© Lungsod at County ng San Francisco`, it looks like there's enough room to put `Francisco` up with `San`. I would prefer if that whole string on the left side was on one line.

**Comment by christine-sfg** [comment]
Do you mean the wrapping of the left column (copyright)? Here's a review app https://dahlia-web-full-pr-1632.herokuapp.com/tl/listings/a0W4U00000JqTLIUA3?react=true

**Comment by emilyjablonski** [comment]
@christine-sfg This is ready for your eyes again! For some reason it looks like the deploy preview did not update after the last change

**Comment by christine-sfg** [comment]
Bummer the workaround isn't as smart as the normal review apps deployments. I actually had to add the `heroku-review-app` label earlier to get it created.

Saved time this time around and just tested on my local with the latest. Looks great!

**Review by christine-sfg — APPROVED** [review]

**Pull request merged**